### PR TITLE
feat: add archived class indicator

### PR DESCRIPTION
**Pull request by carissa-tang**
## Notion ticket link

[Add archived class indicator](https://www.notion.so/uwblueprintexecs/Add-archived-class-indicator-cda2368c0b7b46d4b8a2c2996f2a81db?pvs=4)

## Implementation description
* Add archived indicator for archived classes
image

## Steps to test
1. Visit an archived class and verify that the tag exists and looks as expected
2. Visit an active class and verify that the tag does not exist

## What should reviewers focus on?
* 

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by jfdoming — COMMENTED** [review]
Nice! The spacing between the "archived" notice and the adjacent UI elements seems a bit off to me. Since the designs were more of a quick mock up, wondering if you've played with the spacing at all? Or if you disagree?

**Comment by carissa-tang** [comment]
how's this? @jfdoming 

image

**Comment by jfdoming** [comment]
> how's this? @jfdoming
> 
> image

looks great imo!

File: frontend/src/components/pages/teacher/DisplayClassroomPage/index.tsx [hunk 1]
```diff
@@ -156,6 +163,20 @@ const DisplayClassroomsPage = () => {
         page={1}
         setPage={(newPage) => !newPage && history.push(Routes.CLASSROOMS_PAGE)}
       />
+      {!isActive && (
+        <HStack
+          border="1px solid"
+          borderColor="grey.200"
+          borderRadius="8px"
+          my="8px"
```
Comment (jfdoming) [inline frontend/src/components/pages/teacher/DisplayClassroomPage/index.tsx]: nit: instead of px, maybe chakra units?

**Review by jfdoming — APPROVED** [review]

**Pull request merged**